portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,18 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 900px;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+.standard-size {
+  width: 500px;
+}
+
+.image-on-center{
+  margin-left: 200px;
```

Comment:
It looks like you're trying to center the image. Hard coded values will never adjust so if the user make their window smaller than 200px, there will still be a 200px gap and they'll only see white. What you want to do is use percentage https://www.w3schools.com/howto/howto_css_image_center.asp
However, I don't think that's important enough to block this submission. Great job!